frontend/src/metabase/admin/tasks/containers/BugReportApp.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { Box } from "grid-styled";
+import AdminHeader from "metabase/components/AdminHeader";
+import { t } from "ttag";
+import { UtilApi } from "metabase/services";
+
+function navigatorInfo() {
+  return {
```

Comment:
You could use `_.pick(navigator, "appVersion", "language", ...)` but I don't have a strong preference.